cypress/integration/seededFlows/adminFlows/users/navigateTabs.spec.js

```diff
@@ -0,0 +1,41 @@
+describe('Navigate User Tabs', () => {
+  describe('As an admin', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/adminUser.json').as('user');
+      cy.get('@user').then((user) => {
+        cy.loginAndVisit(user, '/admin/users/2');
+      });
+    });
+
+    it(`navigates to the "Overview" tab`, () => {
+      cy.findByRole('link', { name: /Overview/i }).should('exist');
+      cy.findByRole('link', { name: /Overview/i }).click();
+      cy.url().should('not.contain', 'tab=');
```

Comment:
```suggestion
      cy.findByRole('navigation', { name: 'Member details' }).within(() => {
        cy.findByRole('link', { name: /Overview/i }).should('exist');
        cy.findByRole('link', { name: /Overview/i }).click();
        cy.url().should('not.contain', 'tab=');
      });
```

I think the Cypress test is failing because there's also the "Overview" link in the sidebar - I think this should fix it!

Reply:
Thank you for this! I just updated the E2E test to reflect this. 😊